mongodb/trigger/eventlistener/descriptor.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "mongodb-eventlistener",
+    "type": "flogo:trigger",
+    "title": "mongodb-eventlistener",
+    "version": "1.0.0",
+    "author": "TIBCO Software Inc.",
+    "settings": [
+      {
+        "name": "mongodbConnection",
+        "type": "object",
+        "required": true
+      }
+    ],
+    "handler": {
+      "settings": [
+        {
+          "name": "collection",
+          "type": "string",
+          "required": false
+        },
+        {
+          "name": "listenInsert",
+          "type": "boolean",
+          "required": true,
+          "value": true
+        },
+        {
+          "name": "listenUpdate",
+          "type": "boolean",
+          "required": true,
+          "value": true
+        },
+        {
+          "name": "listenRemove",
+          "type": "boolean",
+          "required": true,
+          "value": true
+        }
+      ]
+    },
+    "outputs": [{
+      "name": "Output",
+      "type": "object"
+    }],
+    "actions": [
```

Comment:
What is this?

Reply:
You mean the name Output which is of type object? This is the placeholder for output schema for our Connector.

Reply:
@balajivemo : There should not be any "actions" in descriptor. Let's remove that.